python/katana/local/_graph.pxd

```diff
@@ -12,7 +12,7 @@ cdef _convert_string_list(l)
 #
 # Python Property Graph
 #
-cdef class PropertyGraphBase:
+cdef class GraphBase:
```

Comment:
interesting to see the differences; like in Java would always be AbstractSOMTHING

Reply:
I was not really following a convention here. I should have I guess. I don't know what the Python convention is, or if there is a Python convention. Python does not have the well known and well defined conventions that Java does.